ChangeLog.md

```diff
@@ -1,5 +1,151 @@
 # Changelog for haskell-language-server
 
+## 1.4.0
+
+After a month of vacation a new hls release has arrived:
+
+- Support for ghc 8.10.6 and 8.10.7
+- The ormolu formatter plugin works with ghc 9.0.1
+- We have a *new brand plugin to let you rename indentifiers* thanks to @OliverMadine's great work
+![rename-demo](https://user-images.githubusercontent.com/30090176/133071923-b3d91b2a-619f-4151-ab76-ca50442e5ab4.gif)
+- *Call hierarchy plugin has been improved* thanks to @July541:
+  - Add call from type signature
+  - Add call from a function pattern
+  - Go to typeclass instance directly
+- We have new docs thanks to @michaelpj: <https://haskell-language-server.readthedocs.io>
+- Now you can ask the executable for included plugins with: `haskell-language-server --list-plugins`
+- *DEPRECATION NOTICE*: after this release we will drop support for ghc versions: 8.6.4, 8.10.3 and 8.10.4
+  - The advise is upgrade ghc to the last minor version: 8.6.5 or 8.10.7
```

Comment:
How about moving this notice to above so that it is easy-to-find?

Reply:
Hmm yeah it needs more visibility but i think it is better to start with good news, what about give it a dedicated header?

Reply:
Sounds good for me

Reply:
done!